feat(Pipelines): Support multiple ints in pipeline

cheikhgwane opened the pull request:
Add support for  multiple ints parameter in pipeline.

## Changes

If relevant, please add some screenshots or a short video.

qgerome commented on src/workspaces/features/RunPipelineDialog/ParameterField.tsx:
```diff
@@ -20,10 +20,29 @@ const ParameterField = (props: ParameterFieldProps) => {
-      if (parameter.type === "int" && value !== "") {
-        onChange(parseInt(value, 10));
+
+      if (parameter.type === "int") {
```
I think it's better to always take the value of the input instead of trying to deal with the int value and the string value.

qgerome commented:
This component is a good candidate for some quick tests

qgerome commented on src/workspaces/features/RunPipelineDialog/RunPipelineDialog.tsx:
```diff
@@ -102,7 +129,18 @@ const RunPipelineDialog = (props: RunPipelineDialogProps) => {
       for (const parameter of version.parameters) {
         const val = fields[parameter.code];
         if (parameter.type === "int") {
```
@cheikhgwane I think we often said it but you do have to handle the case of the float as well.

pvanliefland commented:
@cheikhgwane Quentin has reworked part of this PR.

Could you review the changes, and test it throughly in your dev environment?

Thanks!

cheikhgwane commented:
> @cheikhgwane Quentin has reworked part of this PR.
> 
> 
> 
> Could you review the changes, and test it throughly in your dev environment?
> 
> 
> 
> Thanks!

Yes, on it.

cheikhgwane commented:
@qgerome I've updated the PR to display the parameters values for textarea, and also changed the multiple int param test to use userEvent.paste instead of type (type was returning the value one by one).
